[DX-1250] Added note for Redis Rate Limiter enhancement (#4396)
TykTechnologies/tyk-docs: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md

We have made some changes to the Tyk OAS API Definition to provide a stable contract that will now be under breaking-change control for future patches and releases as Tyk OAS moves out of Early Access. Changes include the removal of the unnecessary `slug` field and simplification of the custom plugin contract.

</details>
</li>
<li>
<details>
<summary>Optimised Gateway memory use and network interaction when using Redis Rate Limiter</summary>

> We have optimised the allocation behaviour of our sliding window log rate limiter implementation ([Redis Rate Limiter](getting-started/key-concepts/rate-limiting#redis-rate-limiter)). Previously the complete request log would be retrieved from Redis but with this enhancement only the count of the requests in the window is retrieved, optimising the network interaction with Redis and decreasing the memory in use on gateway side.

Author: Just following the direction not to include `.md` in links... :)

> We have optimised the allocation behaviour of our sliding window log rate limiter implementation ([Redis Rate Limiter](getting-started/key-concepts/rate-limiting#redis-rate-limiter)). Previously the complete request log would be retrieved from Redis but with this enhancement only the count of the requests in the window is retrieved, optimising the network interaction with Redis and decreasing the memory in use on gateway side.

Author: ```suggestion
We have optimised the allocation behaviour of our sliding window log rate limiter implementation ([Redis Rate Limiter]({{< ref "getting-started/key-concepts/rate-limiting#redis-rate-limiter" > }})). Previously the complete request log would be retrieved from Redis. With this enhancement only the count of the requests in the window is retrieved, optimising the interaction with Redis and decreasing the Gateway memory usage.
```

I don't like splitting this into two sentences, but I'm not going to die on that hill...
